jdbc-driver/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'com.github.johnrengelman.shadow' version '6.0.0'
+}
+
+group 'com.google.cloud.dataproc'
+version '1.0-SNAPSHOT'
+
+// Dataproc (and the Hadoop ecosystem in general) still use Java 8
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile project(path: ':hive', configuration: 'shadow')
+    implementation 'com.google.guava:guava:29.0-jre'
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    testImplementation 'com.google.truth:truth:1.0.1'
+    testImplementation 'com.google.truth.extensions:truth-java8-extension:1.0.1'
```

Comment:
Is it possible to make Gradle inherit dependency versions from the root project? If no, we need to declare versions as properties and use in all modules - this will simplify dependencies upgrade in the future.